### ISSUE TITLE
Adapt popup-width to key-width

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#1044]
The popup-widths were adapted to the regular keyboard. They don't automatically change their width for other keyboards though.

(See pictures at the end)

1. This isn't optimal
2. On the number keyboard, ALL popups have the wrong width, not just the horizontal ones.
3. From a design-standpoint I'm not sure what is the best course of action. Fat popups aren't optimal, but they seem like the best option we have right now.

Upright:
![Screenshot_20210628-112753.jpg](https://user-images.githubusercontent.com/59611881/123615433-961f4580-d805-11eb-8b43-ecf2b142ddea.jpg)

Horizontal:
![Screenshot_20210628-112810.jpg](https://user-images.githubusercontent.com/59611881/123615480-a2a39e00-d805-11eb-8c60-c3f8d348cb46.jpg)

**Comment by Yurieaijoker** [comment]
It would be nice to have an option to disable pop up in separate number keys like in gboard. It just changes the color of  background of the key. Photo attached.
![Uploading Screenshot_20220213_170620.jpg…]()

**Comment by z0xrv** [comment]
@patrickgold

**Comment by patrickgold** [comment]
Due to having to make an rc04 release because of another clipboard crash I figured why not finally fix this issue as well. I had a few different runs locally and decided on the following solution:

In numeric (above screenshot), no key has any popups anymore, this aligns with other keyboard's behaviors now.

In numeric advanced (the layout seen in #1667), the following keys do never show a popup anymore:
- Keys 0 to 9
- Space bar
- Delete key
- Enter key (still has it's extended form like in characters)
- Switch to characters/symbols

The other keys have the popup still, mostly because there are extended popups assigned and I want people to discover them through the three dots indicator. However the keys remaining to have a popup are much smaller and thus I let the popup width as is for these keys. Note that it is certainly possible for the `+`, `(` and `)` to adopt to the key width, but it looks ugly in the extended popups then, so I decided against it.

I hope this fixes the issue (commit linked above), will be released now in rc04 and (hopefully) on Tuesday in 0.3.14 stable, if not another clipboard crash destroys my schedule...
